tests/Exceptionless.Tests/appsettings.yml

```diff
@@ -1,7 +1,11 @@
 ﻿---
 ConnectionStrings:
-  Redis.off: localhost,abortConnect=false
-  Elasticsearch.off: http://localhost:9200/
+  elasticsearch.off: server=http://localhost:9200
+  cache.off: provider=redis;localhost,abortConnect=false
+  messagebus.off: provider=redis;localhost,abortConnect=false
```

Comment:
This is not a valid connection string format. It would have to be `provider=redis;server="localhost,abortConnect=false"`

Reply:
fixed